.github/workflows/go-test.yml

```diff
@@ -0,0 +1,55 @@
+# File managed by web3-bot. DO NOT EDIT.
+# See https://github.com/protocol/.github/ for details.
+
+on: [push, pull_request]
+name: Go Test
+
+jobs:
+  unit:
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ "ubuntu", "windows", "macos" ]
+        go: [ "1.16.x", "1.17.x" ]
```

Comment:
We're bumping from (1.15, 1.16) to (1.16, 1.17). We need to bump go-libp2p, which will bump quic-go or we'll get failures here.